### PR TITLE
work on ReviewsComponent, added apiKey method for server

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/QandA/QandA.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-class App extends React.Component {
+class QandA extends React.Component {
```
Comment (jameslesliecarter) [inline src/QandA/QandA.jsx:3]: Not sure what's going on here with file rename.

**Review by jameslesliecarter — COMMENTED** [review]
Unclear exactly what's going on here. A little clarification please.

**Comment by jameslesliecarter** [comment]
Also looks like we have a lot of merge conflicts with this PR. Please investigate and resubmit.

**Comment by ghost** [comment]
nop.

**Pull request closed without merging**